### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by mihaimyh** [fgilde/MudBlazor.Extensions#3]
I followed the readme trying to use the dialog extensions but I cannot get it to work.
Should I also add your js and css files in my _Host.cshtml / index.html?

**Comment by fgilde** [comment]
You can but this is not required. If not loaded styles and js will automatically loaded.
You need to use die extension method `ShowEx` for the IDialogService

```C#       
var optionsEx = new DialogOptionsEx
        {
            CloseButton = true,
            MaxWidth = MaxWidth.Medium,
            FullWidth = true,
            DisableBackdropClick = true,
            MaximizeButton = true,
            DragMode = MudDialogDragMode.Simple,
            Position = await PositionBasedOnMouse(),
            Animations = new []{AnimationType.SlideIn},
            FullHeight = true,
            DisableSizeMarginY = true,
            DisablePositionMargin = true
        };
 
        return await dialogService.ShowEx<TDialog>(title, parameters, optionsEx);

**Comment by fgilde** [comment]
Hope this helps if not let me know

**Comment by mihaimyh** [comment]
Still doesn't work, I don't see a maximize button, nor the animations.

![image](https://user-images.githubusercontent.com/549272/181859818-2cf50364-c826-4305-87e4-b57cb4316537.png)

![image](https://user-images.githubusercontent.com/549272/181859840-6c8eff83-89fa-4dd4-8e3a-fc5000a44d65.png)

**Comment by fgilde** [comment]
Please pass the option for jsruntime as well then it should work

**Comment by fgilde** [comment]
Works now?

**Comment by mihaimyh** [comment]
I changed it with this one, but no luck, still not working:

```
    var optionsEx = new DialogOptionsEx
    {
        CloseButton = true,
        MaxWidth = MaxWidth.False,
        FullWidth = true,
        DisableBackdropClick = true,
        MaximizeButton = true,
        DragMode = MudDialogDragMode.Simple,
        Animations = new[] { AnimationType.Bounce },
        FullHeight = true,
        DisableSizeMarginY = true,
        DisablePositionMargin = true,
        Resizeable = true,
        JsRuntime = JsRuntime
    };

var dialog = await DialogService.ShowEx<JobRunLogDialog>("Job Log", parameters, optionsEx);
```

**Comment by fgilde** [comment]
Ok that's strange. Do you have any errors in browser console? If you want we can also have a look on it together with teams or you share your repo.

**Comment by falsharif86** [comment]
Having the same issue here

**Comment by falsharif86** [comment]
Worked by passing in Jsruntime

**Comment by mihaimyh** [comment]
It now works for me also by injecting JsRuntime, but there seems to be issues when a displaying a dialog on top of another dialog.
The buttons of the last (child) dialog are not visible.

**Comment by mihaimyh** [comment]
It seems the buttons of the child dialogs are on the same line with the parent dialog buttons

![image](https://user-images.githubusercontent.com/549272/184168579-2072f8d1-2377-46b0-a07d-00cbe294d000.png)

**Comment by fgilde** [comment]
I will check

**Comment by fgilde** [comment]
This bug is solved in 1.4.2
